sys/mips/malta/malta_machdep.c

```diff
@@ -317,7 +355,16 @@ platform_start(__register_t a0, __register_t a1,  __register_t a2,
 			printf("%s ", (char*)(intptr_t)argv[i]);
 		printf("\n");
 	}
-
+	/*
+	 * parse the command line arguments passed e.g. by QEMUs -append option
+	 *
+	 * TODO: start parsing at index 1? QEMU passes -kernel option as
+	 * argv[0] so we currently call kern_setenv("/path/to/kernel", "1")
+	 * Or do other loaders/emulators pass arguments starting at index 0?
+	 */
+	for (int i = 0; i < argc; i++) {
+		_parse_bootargs((char*)(intptr_t)argv[i]);
```

Comment:
I assume that it is guaranteed that on MALTA, the arguments will always be in writable memory, so strsep() on the original rather than a copy is OK?